packages/ra-core/src/controller/list/useList.ts

```diff
@@ -154,7 +154,7 @@ export const useList = <RecordType extends RaRecord = any>(
     useEffect(
         () => {
             if (isLoading || !data) return;
-
+            console.log({ data });
```

Comment:
please don't commit console.log() lines

Reply:
All have been removed - WIP sorry, the last commit was a handover during a remote peer session with @djhi...